doc/api_repos.rst

```diff
@@ -83,10 +83,28 @@ Repository Configuration
 
      List of URLs for this repository. Defaults to ``[]``.
 
+  .. attribute:: cost
+
+    The relative cost of accessing this repository, defaulting to 1000. This value is compared
+    when the priorities of two repositories are the same. The repository with *the lowest cost* is
+    picked. It is useful to make the library prefer on-disk repositories to remote ones.
+
+  .. attribute:: excludepkgs
+
+    List of packages specified by a name or a glob. DNF will exclude every package in the repository
+    that does match this list from all operations. Defaults to ``[]``.
+
   .. attribute:: id
 
     ID of this repo.
 
+  .. attribute:: includepkgs
+
+    List of packages specified by a name or a glob. DNF will exclude any package in the repository
```

Comment:
I think removing `a` is better here.. because we are talking about some real name/glob rather that about something abstract